compiler/tflchef/core/src/CustomOp/AddV2.cpp

```diff
@@ -29,7 +29,7 @@ AddV2Chef::custom_value(flatbuffers::FlatBufferBuilder &fbb) const
 {
   auto &operation = (*_operation);
 
-  assert(operation.type() == "AddV2");
+  assert(operation.custom_code() == "AddV2");
```

Comment:
I thought there will be `type` check as `CustomOp`.
Draft doesn't do the check. Am I missing something?

Reply:
Q) what happens when user missed to add `custom_code` in the `.recipe` file?

Reply:
@seanshpark 
Thank you for comments! 
> Q) what happens when user missed to add custom_code in the .recipe file?

Perhaps, if the user forgets to add `custom_code` in recipe file, it will be caught in the assert code... 😅
What do you think about changing this part to a code that checks with `custom_type` if there is `extype` and `type` if there is no `extype`?

Reply:
logic I think would be is described in https://github.com/Samsung/ONE/pull/11750#discussion_r1368165801 . `extype` or `type` check if `CustomOp` would be `2/` .

Reply:
> it will be caught in the assert code... 

which one? please point out in DRAFT

Reply:
>  if there is extype and type if there is no extype?

I'm not sure how this question is about my `Q) what happens when user missed ... `
Can you come again?